plugins/com.google.cloud.tools.eclipse.sdk.test/src/com/google/cloud/tools/eclipse/sdk/CloudSdkManagerTest.java

```diff
@@ -39,4 +44,63 @@ public void testManagedSdkOption_featureForced() {
     CloudSdkManager.forceManagedSdkFeature = true;
     assertTrue(CloudSdkManager.isManagedSdkFeatureEnabled());
   }
+
+  @Test
+  public void testPreventModifyingSdk_cannotWrite() throws InterruptedException {
```

Comment:
Are these going to work if the tests run in parallel? Should this class be marked @Sequential?

Reply:
Yes! A lot of the tests in this PR won't work in parallel. But I cannot find the `@Sequential` annotation. What's the full package qualification?